lib/member_page.rb

```diff
@@ -0,0 +1,22 @@
+require 'scraped'
+
+class MemberPage < Scraped::HTML
+  field :name do
+    noko.xpath('//div[@class="col-sm-12 col-md-2 distrito"]/following-sibling::div[@class="col-sm-12 col-md-4"]/h1/text()')
```

Comment:
This and the selectors below seem to be very specific. Are the bootstrap classes (`col-*`) actually _required_ for this to work?